integration-tests/src/test/java/oracle/kubernetes/operator/utils/Operator.java

```diff
@@ -17,6 +17,16 @@
 /** Operator class with all the utility methods for Operator. */
 public class Operator {
 
+  public static enum RESTCertType {
+    /*self-signed certificate and public key stored in a kubernetes tls secret*/
+    SELF_SIGNED,
+    /*Certificate signed by an auto-created CA signed by an auto-created root certificate,
+     * both and stored in a kubernetes tls secret*/
+    CHAIN,
+    /*Certificate and public key, and stored in a kubernetes tls secret*/
```

Comment:
When I read the comment for LEGACY RESTCertType this does not seem to be correct in use of TLS secret type?

I believe that LEGACY indicates the test would be using the Helm chart to pass the actual values and then only the private key is kept a secret, is that correct?

Reply:
Yeah, the comment is wrong, it seems like a typical copy/paste issue from the above. Will change to   /*Certificate and public key are stored in a helm values file*/